lib/brainstem/presenter_collection.rb

```diff
@@ -275,7 +278,16 @@ def run_search(scope, includes, sort_name, direction, options)
         search_options[:page] = calculate_page(options)
       end
 
-      search_options.reverse_merge!(extract_filters(options))
+      extracted_filters = extract_filters(options)
```

Comment:
@cantino: I switched to setting the `filters_hash` to `{ arg: arg, include_params: filter_options[:include_params] }`, like you mentioned. The one tricky part is making sure that `run_search` gets the extracted filters in the format it wants, which is what I'm doing here.